### ISSUE TITLE
RuntimeWarning: invalid value encountered in divide during example training

### ISSUE THREAD

**Issue by nykh** [wkentaro/pytorch-fcn#37]
In the first round of evaluation (with pretrained VGG16 model) there is following error 
```console
.../pytorch-fcn/examples/voc/torchfcn/utils.py:24: RuntimeWarning: invalid value encountered in divide
  acc_cls = np.diag(hist) / hist.sum(axis=1)                                    
.../pytorch-fcn/examples/voc/torchfcn/utils.py:26: RuntimeWarning: invalid value encountered in divide
  iu = np.diag(hist) / (hist.sum(axis=1) + hist.sum(axis=0) - np.diag(hist))
```
and the training loss becomes `NaN`. Is this expected? 

Also I am not sure if it's related but AFTER the first round all the validation error becomes `NaN`.

**Comment by wkentaro** [comment]
It's expected behavior.
FYI, you can ignore the message by export PYTHONWARNINGS=ignore
